packages/frontend-core/src/components/grid/overlays/KeyboardManager.svelte

```diff
@@ -58,7 +58,7 @@
         case "c":
           return handle(() => dispatch("copy"))
         case "v":
-          return handle(() => dispatch("paste"))
+          return dispatch("paste")
```

Comment:
NAB: Why is this different to the "copy" case?

Reply:
It's because we want to enable the native browser paste. If we wrap this in `handle` then that will `preventDefault` on the event, meaning it stops the browser native paste from ever working. This was what used to happen before this PR.

Copy is different because we already put the copied value into the clipboard ourselves, so we don't rely on the native copy functionality. Pasting is different as you cannot forcefully read from the clipboard in JS (for security reasons, and at least not without annoying popups), otherwise I would continue to ignore the native paste functionality and just read from the clipboard in JS and handle it ourselves.